Cargo.toml

```diff
@@ -14,15 +14,15 @@ description = "Integration of `amethyst` and `rhusics`"
 keywords = ["gamedev", "amethyst", "physics"]
 
 [dependencies]
-amethyst_core = { version = "0.3.0" }
-amethyst_renderer = { version = "0.8.0" }
+amethyst_core = { version = "0.3", git = "https://github.com/amethyst/amethyst" }
```

Comment:
Will this cause issues when people are editing amethyst locally?

Reply:
wait, for some reason I thought this was inside `amethyst/amethyst`. ignore me 🤦‍♂️

Reply:
Hehe :)

Reply:
These will be removed before it's released (which will happen right after the amethyst release).